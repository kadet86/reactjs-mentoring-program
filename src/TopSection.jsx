import React from 'react';

const TopSection = ({children}) => (
    <section className="top-section">
        <div className="top-section__overlay"></div>
        <div className="top-section__inner-container">
            {children}
        </div>
    </section>
);

export default TopSection;