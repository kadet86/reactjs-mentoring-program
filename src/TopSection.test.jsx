import React from 'react';
import {shallow} from 'enzyme';
import TopSection from './TopSection';

describe('TopSection component', () => {
    it('has persistent snapshot', () => {
        const component = shallow((
            <TopSection>
                <div id="1"></div>
                <div id="2"></div>
            </TopSection>
            )
        );
        expect(component).toMatchSnapshot();
    });
});