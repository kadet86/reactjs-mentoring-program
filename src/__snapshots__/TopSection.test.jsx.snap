// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopSection component has persistent snapshot 1`] = `"<section class=\\"top-section\\"><div class=\\"top-section__overlay\\"></div><div class=\\"top-section__inner-container\\"><div id=\\"1\\"></div><div id=\\"2\\"></div></div></section>"`;
